assets/js/mentors.js

```diff
@@ -2,6 +2,7 @@ var controllerMentors = (function(jQuery) {
     var skillsButton = jQuery('.skills');
     var presentationButton = jQuery('.presentation');
     var menteesButton = jQuery('.mentees');
+    var reviewsButton = jQuery('.reviews');
```

Comment:
```suggestion
    let reviewsButton = jQuery('.reviews');
```

I just remembered about use let instead of var  =D